demos/optical_character_recognition/python/README.md

```diff
@@ -107,22 +107,22 @@ Custom nodes are loaded into OVMS as dynamic library implementing OVMS API from
 It can use OpenCV libraries included in OVMS or it could use other thirdparty components.
 
 The custom node east_ocr can be built inside a docker container via the following procedure:
-- go to the custom node source code folder [src/custom_nodes/east_ocr](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/src/custom_nodes/east_ocr)
-- run `make` command
+- go to the directory with custom node examples [src/custom_node](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/src/custom_nodes)
+- run `make` command:
 
 ```bash
 git clone https://github.com/openvinotoolkit/model_server.git
-cd model_server/src/custom_nodes/east_ocr
-make
-cd ../../../../
+cd model_server/src/custom_nodes
+make NODES=east_ocr
```

Comment:
Why `BASE_OS` is specified in other `make` calls, but not here?

Reply:
By mistake, I will add it